tap_jira/streams.py

```diff
@@ -140,6 +141,48 @@ def sync(self):
                 for page in pager.pages(COMPONENTS.tap_stream_id, "GET", path):
                     COMPONENTS.write_page(page)
 
+    def sync_cloud(self):
+        offset = 0
+        while True:
+            params = {
```

Comment:
If the inner code is the same then can we reuse the code of def sync_on_prem?

Reply:
I can see redundant codes so we can make a seprate function as well and call after request call.